bin/mpi_select_mock_targets

```diff
@@ -36,7 +36,6 @@ parser.add_argument('--realtargets', '-r', help='Path to real target catalog', t
 parser.add_argument('-v','--verbose', action='store_true', help='Enable verbose output.')
 parser.add_argument('--no-check-env', action='store_true', help="Don't check NERSC environment variables")
 parser.add_argument('--sort-pixels', action='store_true', help="Sort pixels by galactic latitude")
-parser.add_argument('--no-spectra', action='store_true')
```

Comment:
For the record, from an email with @forero: `select_mock_targets --no-spectra` is now fast enough that we don't necessarily need an MPI wrapper for it.  We may want to re-enable this feature in the future to enable running the full survey in a few minutes, but ok for this to be disabled for now.